src/store.ts

```diff
@@ -1,11 +1,7 @@
 import { Injectable, ErrorHandler } from '@angular/core';
-import { Observable } from 'rxjs/Observable';
-import { Subscription } from 'rxjs/Subscription';
-import { distinctUntilChanged, catchError, take } from 'rxjs/operators';
-import { forkJoin } from 'rxjs/observable/forkJoin';
-import { map } from 'rxjs/operators/map';
-import { fromPromise } from 'rxjs/observable/fromPromise';
-import { of } from 'rxjs/observable/of';
+import { Observable, Subscription, forkJoin, of } from 'rxjs';
+import { map, distinctUntilChanged, catchError, take } from 'rxjs/operators';
+import { fromPromise } from 'rxjs/internal/observable/fromPromise';
```

Comment:
Is the `internal` really supposed to be used?

Reply:
fromPromise has been moved to `from` so we need to change all fromPromise

Reply:
i see this import pattren [here](https://github.com/ReactiveX/rxjs/blob/master/compat/add/observable/fromPromise.ts)
```ts
import { fromPromise } from 'rxjs/internal-compatibility';
```
`from` is also available  from `rxjs`
```ts
import { Observable, Subscription, forkJoin, of, from  } from 'rxjs';
```
@leon is this `from` can convert to ` Observable` ?